www/next.config.js

```diff
@@ -1,6 +1,568 @@
 const withMDX = require('@next/mdx')()
 
 module.exports = withMDX({
-  basePath: '/new',
+  basePath: '',
   pageExtensions: ['js', 'jsx', 'tsx', 'md', 'mdx'],
+  trailingSlash: false,
+
+  async headers() {
+    return [
+      {
+        source: '/(.*)',
+        headers: [
+          {
+            key: 'Strict-Transport-Security',
+            value: '',
+          },
+          {
+            key: 'X-Robots-Tag',
+            value: 'all',
+          },
+          {
+            key: 'X-Frame-Options',
+            value: 'DENY',
+          },
+        ],
+      },
+    ]
+  },
+
+  async rewrites() {
+    return [
+      {
+        source: '/:path*',
+        destination: `/:path*`,
+      },
+      {
+        source: '/docs',
+        destination: `${process.env.NEXT_PUBLIC_DOCS_URL}`,
```

Comment:
currently set to https://zones-docs.vercel.app should be changed to docs-rose.vercel.app...or production branch for zones-docs needs to be set to master

Reply:
probably going to retire the old project (docs-rose.vercel.app) to make switching over easier